easybuild/easyconfigs/m/mkl-service/mkl-service-2.0.1-intel-2019a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'mkl-service'
+version = '2.0.1'
```

Comment:
(note that version 2.0.2 is out)

Reply:
Yeah, might as well update... done in 3580427fcc